tests/components/multi-audio/multiAudioPopover-test.js

```diff
@@ -1,53 +0,0 @@
-jest.dontMock('../../../js/components/multi-audio/multiAudioPopover');
```

Comment:
Are we missing tests for `multiAudioCloseCaptionScreen` or other new components?

Reply:
\+

Reply:
[Added tests for closedCaptionsMultiAudioMenu component](https://github.com/ooyala/html5-skin/pull/1035/commits/7aad5e2443f372b6745bf3006179956400646d5c)